assets/docs/sayannath/american-sign-language/1.md

```diff
@@ -0,0 +1,8 @@
+# Placeholder sayannath/american-sign-language/1
```

Comment:
This is great!

Do you also want to upload the SavedModel from https://github.com/sayannath/American-Sign-Language-Detection/releases/download/1.0.0/asl_classifier.zip, from which the TFLite model was converted?

Reply:
Cool! I will add the SavedModel as well.

Reply:
Nice! Then, you don't need to provide a config for a `Placeholder` anymore but can simply replace it in the first line with `Module`. https://www.tensorflow.org/hub/writing_documentation#model for reference.